server/paymentProviders/stripe/index.js

```diff
@@ -206,13 +207,20 @@ export default {
     }
   },
 
-  webhook: requestBody => {
+  webhook: request => {
+    const requestBody = request.body;
+
```

Comment:
Would be great to add some optional logging of all received webhook requests, using the debug library.